packages/audits/src/Api/reducer.ts

```diff
@@ -0,0 +1,101 @@
+import { createSlice, createAction, PayloadAction } from '@reduxjs/toolkit';
+import { initialState } from './initialState';
+import { LoadAudits, Filter, DataSorting } from './interfaces';
+import moment from 'moment';
+
+export const defaultItemsPerPage = 20;
+
+const { name: storeName, actions: lifeCycleActions, reducer } = createSlice({
+  name: 'audits',
+  initialState,
+  reducers: {
+    setContext: (state, { payload }) => {
+      state.context = payload;
+    },
+    startLoading: (state) => {
+      state.isLoading = true;
```

Comment:
Don't mutate the store use it like: 

```
startLoading: (state) => ({...state, isLoading: true})
```

Reply:
About mutable.
In the [documentation](https://redux-toolkit.js.org/api/createReducer#direct-state-mutation) is written 
`To make things easier, createReducer uses immer to let you write reducers as if they were mutating the state directly. In reality, the reducer receives a proxy state that translates all mutations into equivalent copy operations.
`

But I will agree with you - the style of code should be one for the whole project.